helm-util/_example.jsonnet

```diff
@@ -0,0 +1,12 @@
+local helm = import 'github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet';
+
+{
+  // render the Grafana Chart at version 5.5.5
+  grafana: helm.template('grafana', 'grafana', {
+    values: {
+      persistence: { enabled: true },
+      plugins: ['grafana-clock-panel'],
+    },
+    flags: ['--version=5.5.5'],
```

Comment:
Oef, old version and from stable repo?

Reply:
5.5.5 is the chart version, not the Grafana one. Whatever that means in the end.

Is stable not correct?

Reply:
Stable is deprecated: https://github.com/helm/charts#deprecation-timeline

Reply:
But we don't have our own repo yet, right?

Reply:
It might not be the best example :yum: